python/common/org/Python.java

```diff
@@ -602,12 +602,44 @@ public static void initializeModule(java.lang.Class cls, java.util.Map<java.lang
                     "iteration.  The enumerate object yields pairs containing a count (from\n" +
                     "start, which defaults to zero) and a value yielded by the iterable argument.\n" +
                     "enumerate is useful for obtaining an indexed list:\n" +
-                    "       (0, seq[0]), (1, seq[1]), (2, seq[2]), ...\n"
+                    "(0, seq[0]), (1, seq[1]), (2, seq[2]), ...\n",
+            args = {"items"},
+            default_args = {"start"}
     )
-    public static org.python.Iterable enumerate() {
-        throw new org.python.exceptions.NotImplementedError("Builtin function 'enumerate' not implemented");
+    public static org.python.Iterable enumerate(org.python.Object items, org.python.Object start) {
+        try {
```

Comment:
This `try` block is quite large, and it's not immediately obvious where the `AttributeError` could come from. I'm guessing it's for `org.Python.iter()`, so to make the code clearer by minimizing the coverage, only that and everything that depends the `iter` should be inside.

I think one way of doing this would be to move the initial index/start handling code out to before the try block, that should be the minimum.